src/univariate/discrete/binomial.jl

```diff
@@ -73,6 +73,24 @@ function mode(d::Binomial{T}) where T<:Real
 end
 modes(d::Binomial) = Int[mode(d)]
 
+function median(dist::Binomial)
+    bound = min(dist.p, 1-dist.p) # from http://dx.doi.org/10.1111/j.1467-9574.1980.tb00681.x
+    dist_mean = mean(dist)
+    
+    floor_mean = floor(Int, dist_mean)
+    difference = dist_mean - floor_mean
+    
+    if difference < bound
```

Comment:
AFAICT we can use equality here:
```suggestion
    if difference <= bound
        # The only case where the median satisfies |median - mean| <= 1 - bound with equality
        # is p = 1/2 and n odd
        # However, in that case we also want to return floor(mean)
```